src/SprykerEco/Zed/Adyen/Business/Hook/Saver/MakePayment/CreditCardSaver.php

```diff
@@ -42,10 +42,17 @@ protected function updatePaymentAdyenTransfer(
     }
 
     /**
+     * @param \Generated\Shared\Transfer\PaymentAdyenTransfer|null $paymentAdyenTransfer
+     *
      * @return string
      */
-    protected function getPaymentStatus(): string
+    protected function getPaymentStatus(PaymentAdyenTransfer $paymentAdyenTransfer = null): string
     {
+        if ($paymentAdyenTransfer) {
+            if ($paymentAdyenTransfer->getResultCode() === $this->config->getOmsStatusRefused())
+             return $this->config->getOmsStatusRefused();
+        }
```

Comment:
Please use early return here. Also we do not use short `if` statement without `{}`.

Reply:
From my perspective I see no way to use the early return in this case. Please, provide some example.